content/en/serverless/configuration/_index.md

```diff
@@ -359,6 +360,12 @@ To parse and transform your logs in Datadog, see documentation for [Datadog log
 
 To see what libraries and frameworks are automatically instrumented by the Datadog APM client, see [Compatibility Requirements for APM][15]. To instrument custom applications, see Datadog's APM guide for [custom instrumentation][16].
 
+## Select sampling rates for ingesting APM spans 
+
+To manage the [APM traced invocation sampling rate][35] for serverless functions, set the `DD_TRACE_SAMPLE_RATE` environment variable on the function you want to change the sampling rate for to a value between 0.000 (trace 0% of Lambda function invocations) and 1.000 (trace all Lambda function invocations).
+
+Setting a sampling rate of 1-10% is recommended to manage usage of traced invocations from your Lambda functions.
```

Comment:
I'm not sure that we really need (or should) give ppl the recommended rate. If we do, probably should give a better (more complex) recommendation, e.g., 100% for cronjob/staging apps, and lower rate for super busy prod functions yielding similar traces.

Reply:
good feedback, updated